__tests__/app.test.js

```diff
@@ -18,16 +19,24 @@ describe('/api/topics', () => {
         .get('/api/topics')
         .expect(200)
         .then(({body}) => {
-            expect(body.topics).toEqual([{
-                  description: 'The man, the Mitch, the legend',
-                  slug: 'mitch'
-                },{
-                  description: 'Not dogs',
-                  slug: 'cats'
-                },{
-                  description: 'what books are made of',
-                  slug: 'paper'
-                }])
+            body.topics.forEach((topic) => {
+                expect(topic).toHaveProperty('description')
+                expect(topic).toHaveProperty('slug')
+            })
+        })
+    });
+});
+describe('/api', () => {
+    test('GET: responds with an object with all endpoints and info about them', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then(({body}) => {
+            return fs.readFile('./endpoints.json')
+            .then((endpoints) => {

```

Comment:
lets not nest .then chains 🙂 
lets consider using promiseAll to write out this test 🙂